rfcs/text/0021-threat-enrichment.md

```diff
@@ -29,6 +24,22 @@ Stage 1: If the changes include field additions or modifications, please create
 Stage 1: Describe at a high level how this change affects fields. Include new or updated yml field definitions for all of the essential fields in this draft. While not exhaustive, the fields documented here should be comprehensive enough to deeply evaluate the technical considerations of this change. The goal here is to validate the technical details for all essential fields and to provide a basis for adding experimental field definitions to the schema. Use GitHub code blocks with yml syntax formatting, and add them to the corresponding RFC folder.
 -->
 
+As these fields represent the enrichment of an existing event with indicator information, they are comprised of three categories of data:
+
+1. The indicator's indicator fields, as defined in RFC 0018
+2. Other relevant ECS fields from the indicator (see below)
+3. Fields representing the context of the enrichment itself
+
+### Proposed new fields
+
+Field | Type | Example | Description
+--- | --- | --- | ---
+matched.atomic | keyword | 2f5207f2add28b46267dc99bc5382480 | The value that matched between the event and the indicator
```

Comment:
Can these fields be written with their "fully qualified" field name here for clarity?

I think `matched.atomic` will be `threat.enrichments.matched.atomic`, but I'd rather that's explicitly captured.